tests/network/macvtap.go

```diff
@@ -87,184 +78,21 @@ var _ = SIGDescribe("Macvtap", decorators.Macvtap, Serial, func() {
 			chosenMACHW, err := GenerateRandomMac()
 			Expect(err).ToNot(HaveOccurred())
 			chosenMAC = chosenMACHW.String()
-			serverCIDR := "192.0.2.102/24"
 
-			serverIP, err = libnet.CidrToIP(serverCIDR)
+			const macvtapNetName = "test-macvtap"
+			serverVMI := libvmi.NewAlpineWithTestTooling(
```

Comment:
The name `serverVMI` doesn't fit anymore since there is no client. I would change to `vmi`.